docs/reference/google_drive.md

```diff
@@ -0,0 +1,96 @@
+
+To get started, log into [Google Cloud Platform](cloud.google.com) and go to the `Console`.
+
+1. **Create a Google Cloud Project.** Give your project a name, change the project ID and click the Create button.
+
+2. **Enable Google APIs.** Choose APIs & Services from the left menu and click on `Enable APIs and Services`. You need to enable the Drive API.
+
+3. **Create a Service Account.** In the `APIs & Services` section, click on `Credentials` and click on `Create credentials` to create a service account. Give your service account a name and a service account ID. This is like an email address and will be used to identify your service account in the future. Click `Done` to finish creating the service account.
+
+4. **Create a Key File**.
+  - In the Cloud Console, go to `IAM and Admin` > `Service accounts` page.
+  - Click the email address of the service account that you want to create a key for.
+  - Click the `Keys` tab. Click the `Add key` drop-down menu, then select `Create new key`.
+  - Select JSON as the Key type and then click `Create`. This will download a JSON file that will contain the service account credentials.
+
+5. **Share Google Drive Folders.** Go to your Google Drive. Right-click the folder or shared drive, choose `Share` and add the email address of the service account you created in step 3. as a viewer to this folder.
+
+  Note: When you grant a service account access to a specific folder or shared drive in Google Drive, it's important to note that the permissions extend to all the children within that folder or drive. This means that any folders or files contained within the granted folder or drive inherit the same access privileges as the parent.
+
+## Configuration
+
+The following configuration fields need to be provided for setting up the connector:
+##### `Google Drive service account JSON`
+The service account credentials generated from Google Cloud Platform (JSON string). Refer to the [Google Cloud documentation](https://developers.google.com/workspace/guides/create-credentials#create_credentials_for_a_service_account) for more information.
+
+## Documents and syncs
+
+The connector will fetch all files and folders the service account has access to.
+
+It will attempt to extract the content from Google Suite documents (Google Docs, Google Sheets and Google Slides) and regular files.
+
+**NOTE**:
+- Files bigger than 10 MB won't be extracted
+- Permissions are not synced. **All documents** indexed to an Elastic deployment will be visible to **all users with access** to that Elastic Deployment.
+
+## Sync rules
+
+[Basic sync rules](https://www.elastic.co/guide/en/enterprise-search/8.9/sync-rules.html#sync-rules-basic "Basic sync rules") are identical for all connectors and are available by default.
+
+Advanced sync rules are not available for this connector in the present version. Currently filtering is controlled via ingest pipelines.
+
```

Comment:
We usually have a Deployment with Docker section for these connectors too

Reply:
Thank you for spotting this, I added the section about Docker